releasenotes/notes/0.45/add-limits-in-commutation-checker-66a1b9e90921621f.yaml

```diff
@@ -1,5 +1,5 @@
 ---
-features:
+features_circuits:
   - |
     Added a new option ``max_num_qubits`` to :meth:`qiskit.circuit.CommutationChecker.commute`
```

Comment:
The `CommutationChecker` class isn't included in the docs builds so this link does work. We need to add an autosummary section for it to: https://github.com/Qiskit/qiskit/blob/main/qiskit/circuit/__init__.py